DOC-2120-query-udf-disabled-after-chang (#295)
tigergraph/gsql-docs: modules/querying/pages/func/query-user-defined-functions.adoc

There are two ways to modify the UDF files to add user-defined functions to GSQL.
Both are secured by `gadmin` and require administrator-level access:

* xref:_use_github_to_store_udfs[Store UDF files in a GitHub repository], and configure GSQL to read from the repository.
** This is the recommended approach.
It also takes the highest precedence.
If GSQL is configured to read from GitHub for UDFs and the configurations are valid, UDFs in other sources are ignored.
* xref:#_upload_udfs_with_gsql_put_command[Use the GSQL `PUT` command to upload UDFs].
** For security reasons, this method is disabled by default, and can only be enabled by the TigerGraph Linux user with access to `gadmin`.

[IMPORTANT]

> .Need to reinstall queries using UDFs

Author: ```suggestion
.Query rebuild needed if UDFs are modified
```

====

> If you update the UDF files, then any existing installed queries which use any UDF, whether modified or not, will be disabled as a precaution.

Author: ```suggestion
If the UDF files are updated, either through `GitHub` or `PUT`, then any existing installed queries which use any UDF, whether modified or not, will be disabled as a precaution.
```

> To restalling them:

Author: ```suggestion
To re-enabled them:
```

> . Check each query to see if it needs to be modified or not.  Make necessary modifications.

Author: ```suggestion
. Check each query to see the UDF change causes a need to change the query.  Make necessary modifications.
```
